config/locales/en.yml

```diff
@@ -295,21 +295,21 @@ en:
             enrichments:
               blank: "^Complete your course information before publishing"
             sites:
-              blank: "^Select at least one location for this course"
+              blank: "^Select at least one location"
             age_range_in_years:
               blank: "^Select an age range"
             program_type:
-              blank: "^Select a program type"
+              blank: "^Select a funding type"
             subjects:
               blank: "^There is a problem with this course. Contact support to fix it (Error: S)"
-              course_creation: "^Select at least one subject"
+              course_creation: "^Select a subject"
               duplicate: "^You have already selected this subject. You can only select a subject once"
             modern_languages_subjects:
               select_a_language: "^Select at least one language"
             study_mode:
               blank: "^Select a study mode"
             applications_open_from:
-              blank: "^Select when applications will open and enter the date if applicable"
+              blank: "^Select an applications open date"
```

Comment:
I don't think we need these `^` characters anymore but that's for another ticket i think